packages/auth-magic-link-nest/src/magic-link/callbackUrl.middleware.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable, NestMiddleware } from '@nestjs/common';
+import { NextFunction, Request, Response } from 'express';
+
+@Injectable()
+export class callbackUrlMiddleware implements NestMiddleware {
+  use(req: Request, res: Response, next: NextFunction): any {
+    const callbackUrl = req.query.callbackUrl;
+    if (!callbackUrl) {
+      req.query.callbackUrl = 'default';
```

Comment:
there should be something more sophisticated than just 'default'. It should be something that cannot be the callbackUrl by normal usage of this plugin.

additionally this string should be extracted as a constant. we have a file called `constants.ts` where such values can be placed.

Reply:
added DefaultCallbackUrl as a constant in constants.ts